roles/edpm_bootstrap/tasks/packages.yml

```diff
@@ -39,7 +39,7 @@
 - name: Install required packages to bootstrap EDPM
   ansible.builtin.dnf:
     name: "{{ edpm_bootstrap_packages_bootstrap }}"
-    state: present
+    state: latest
```

Comment:
this isn't the only place we install packages. Should we make this change consistent across all the tasks files? ``state: latest`` probably needs to be added to all of the download_cache.yml task files as well.

Reply:
Thank you @slagle for reviewing it.
https://github.com/openstack-k8s-operators/ci-framework/pull/936 fixes the issue. But the proper solution would be do a dnf update via edpm_ansible itself. I will update this pr to add the dnf update.

Reply:
`state: latest` shouldn't be used during packages installation as best practice. (That's why there's the `package-latest` linting test)

Having this task in bootstrap role has the happy side effect that the repo cache will be updated during the installation, so i guess that every package installed after that will be their current version. 

I'm wondering if we need to add `update_cache: true` ([link](https://docs.ansible.com/ansible/latest/collections/ansible/builtin/dnf_module.html#parameter-update_cache)) to this task, though.